packages/paste-website/src/components/site-wrapper/site-header/SiteHeaderSearch.tsx

```diff
@@ -22,7 +23,11 @@ const SiteHeaderSearch: React.FC = () => {
     setIsOpen(false);
   };
 
-  useHotkeys("mod+k", onOpen);
+  const keyCombinationState = useKeyCombination({
+    keys: [platformTriggerKey, "k"],
+    onCombinationPress: onOpen,
+    enablePressStyles: true,
```

Comment:
enablePressStyles doesn't trigger the press styles but enables them when the key is pressed. The logic is handled by component and hook but will only show them if this is set to true. By default press styles are not shown which was the desired functionality.

Reply:
when the search modal is open, it feels a little funny to have the press styles enabled in the background. since you're typing into the search input and could use a "k" and see the press styles going in and out in the background. tiny nit, just thought I'd mention!

Reply:
the open state is stored in the same file so I can disable it when the modal it open (which it should do anyway).